Alignment/OfflineValidation/python/TkAlAllInOneTool/configTemplates.py

```diff
@@ -17,6 +17,16 @@
 
 ######################################################################
 ######################################################################
+loadGlobalTagTemplate="""
+#Global tag
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag,".oO[GlobalTag]Oo.")
```

Comment:
Ok if we're already doing it the preferred way we should keep it.  I thought this looks like an alternative way of doing the same thing:
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Configuration/AlCa/python/GlobalTag.py#L102